library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -17,14 +17,38 @@ namespace Extensions {
 namespace HttpFilters {
 namespace PlatformBridge {
 
+static void envoy_filter_release_callbacks(const void* context) {
```

Comment:
nit: I wonder if we should move the static helpers to their own utility file.

Reply:
Well, they're static, so they're not accessible ouside this file. And we don't need them to be, since they're just what we use to set the function pointers on the struct.

Reply:
We could include them in a bare header, but I suppose my preference is to put them here, to limit their visibility strictly to this file.